scalafmt-core/shared/src/main/scala/org/scalafmt/config/XmlLiterals.scala

```diff
@@ -0,0 +1,15 @@
+package org.scalafmt.config
+
+import metaconfig._
+
+case class XmlLiterals(
+    assumeFormatted: Boolean = false
```

Comment:
may be true by default?

Reply:
if we keep the flag, i'd set it to false, just like we did with `docstrings.wrap` and `comments.wrap`, they are logically similar, i think.